src/core/Mixer.cpp

```diff
@@ -725,30 +732,30 @@ void Mixer::removePlayHandle( PlayHandle * _ph )
 		// (See tobydox's 2008 commit 4583e48)
 		if ( removedFromList )
 		{
-			if( _ph->type() == PlayHandle::TypeNotePlayHandle )
+			if( ph->type() == PlayHandle::TypeNotePlayHandle )
 			{
-				NotePlayHandleManager::release( (NotePlayHandle*) _ph );
+				NotePlayHandleManager::release( (NotePlayHandle*) ph );
 			}
-			else delete _ph;
+			else delete ph;
```

Comment:
Need to add explicit braces.
```suggestion
			else { delete ph; }
```